editions/tw5.com/tiddlers/saving/Saving via WebDAV.tid

```diff
@@ -24,4 +24,4 @@ Many [[NAS|https://en.wikipedia.org/wiki/NAS]] or [[Subversion|https://en.wikipe
 
 !! Free Hosting
 
-None of the ~WebDAV hosting services tested (~~box.com~~, ~~swissdisk.com~~) support accessing HTML files via a webbrowser, so they won't work with ~TiddlyWiki. Maybe someday soon there will be a free ~WebDAV hosting service for ~TiddlyWikis.
+Several ~WebDAV hosting services tested (~~box.com~~, ~~swissdisk.com~~) don't support accessing HTML files via a webbrowser, so they won't work with ~TiddlyWiki. However, The GMX mediacenter (www.gmx.net) has been reported as working with WebDAV with a free account.
```

Comment:
According to [this thread:](https://groups.google.com/forum/#!searchin/tiddlywiki/seafile%7Csort:date/tiddlywiki/QduSiYUX74U/aQqd6LV_AgAJ) seafile should also work with WebDav. ... I didn't test it myself.

Reply:
seafile doesn't appear to be a service. It appears to be a Webdav (and other) client/server software package that you run on your own equipment. None of the pricing mentions volume of data, which would be important for an actual cloud service.